spec/grape/middleware/stack_spec.rb

```diff
@@ -35,8 +35,7 @@ def initialize(&block)
       expect { subject.use StackSpec::BarMiddleware, false, my_arg: 42 }
         .to change { subject.size }.by(1)
       expect(subject.last).to eq(StackSpec::BarMiddleware)
-      expect(subject.last.args).to eq([false])
-      expect(subject.last.opts).to eq(my_arg: 42)
+      expect(subject.last.args).to eq([false, {my_arg: 42}])
```

Comment:
Rubocop will complain, add some spaces in `{ }`.

Reply:
Will fix.
BTW, I'm getting lots of error from RuboCop, I'm not sure why that is:
```
An error occurred while Layout/LineLength cop was inspecting /home/eregon/code/grape/lib/grape/middleware/formatter.rb:100:10.
To see the complete backtrace run rubocop -d.
An error occurred while Layout/LineLength cop was inspecting /home/eregon/code/grape/lib/grape/middleware/formatter.rb:102:17.
To see the complete backtrace run rubocop -d.
An error occurred while Layout/LineLength cop was inspecting /home/eregon/code/grape/lib/grape/middleware/formatter.rb:117:12.
To see the complete backtrace run rubocop -d.
...
```

Reply:
Fixed

Reply:
Strange, dunno why rubocop is having trouble.